pkg/operator/ceph/cluster/mgr/orchestrator.go

```diff
@@ -40,12 +40,23 @@ func (c *Cluster) configureOrchestratorModules() error {
 		return nil
 	}
 
+	if err := client.MgrEnableModule(c.context, c.Namespace, rookModuleName, true); err != nil {
```

Comment:
Can we enable modules asynchronously too?

Reply:
I don't think we'll gain much here. Enabling the rook orchestration (line 49) requires that the rook module here is already enabled. So if we do it in parallel, the command below will just have to wait for it anyway.